eql/track.json

```diff
@@ -102,6 +103,7 @@
         "name": "sequence_2stage_equalityFilter_maxspan1m_fetch1000_size500_slow",
         "operation-type": "eql",
         "request-timeout": 3600,
+        "cluster" : "{{cluster | default('')}}",
```

Comment:
Instead of setting the default in several places we could specify a j2 env var at the top, like so:

```
diff --git a/eql/track.json b/eql/track.json
index 0740a9d..264cfdd 100644
--- a/eql/track.json
+++ b/eql/track.json
@@ -1,4 +1,5 @@
 {% import "rally.helpers" as rally with context %}
+{% set cluster = cluster | default('') %}
 {
   "version": 2,
   "description": "EQL benchmarks based on endgame index of SIEM demo cluster",
@@ -86,7 +87,7 @@
         "name": "sequence_2stage_nofilter_fetch1000_size_1000",
         "operation-type": "eql",
         "request-timeout": 3600,
-        "cluster" : "{{cluster | default('')}}",
+        "cluster" : "{{cluster }}",
         "index": "endgame-4.28*",
         "body": {
           "query": "sequence by source.ip, destination.ip [process where true] [process where true]",
@@ -103,7 +104,7 @@
         "name": "sequence_2stage_equalityFilter_maxspan1m_fetch1000_size500_slow",
         "operation-type": "eql",
         "request-timeout": 3600,
-        "cluster" : "{{cluster | default('')}}",
+        "cluster" : "{{cluster }}",
         "index": "endgame-4.28*",
         "body": {
           "query": "sequence by source.ip, destination.ip with maxspan=1m [process where user.name != \"SYSTEM\"] [process where user.name == \"SYSTEM\"]",
@@ -120,7 +121,7 @@
         "name": "sequence_2stage_nofilter_fetch1000_size1000_tail1000_head200",
         "operation-type": "eql",
         "request-timeout": 3600,
-        "cluster" : "{{cluster | default('')}}",
+        "cluster" : "{{cluster }}",
         "index": "endgame-4.28*",
         "body": {
           "query": "sequence by source.ip, destination.ip [process where true] [process where true] | tail 1000 | head 200",
@@ -154,7 +155,7 @@
         "name": "sequence_3stage_startsWithfilter_maxspan30m_fetch1000_size100_tail100_head50",
         "operation-type": "eql",
         "request-timeout": 3600,
-        "cluster" : "{{cluster | default('')}}",
+        "cluster" : "{{cluster }}",
         "index": "endgame-4.28*",
         "body": {
           "query": "sequence by host.name with maxspan=30m [process where startsWith(process.name, \"ssh\") == true] by process.ppid [process where startsWith(process.name, \"ssh\") == false] by process.ppid [process where startsWith(process.name, \"ssh\") == true] by process.pid | tail 100 | head 50",
```